ui/app/index.html

```diff
@@ -1,32 +1,32 @@
 <!DOCTYPE html>
 <html lang="en" ng-app="am">
 <head>
-	<script src="/lib/jquery.min.js"></script>
```

Comment:
I'm not a fan of relative paths either, but the alternative would be to introduce Go templating into the asset serving in order to be able to inject the path prefix. See https://github.com/prometheus/alertmanager/issues/212.